packages/headless/src/controllers/insight/user-actions/headless-user-actions.ts

```diff
@@ -80,6 +85,10 @@ export function buildUserActions(
     fetchUserActions(userId: string) {
       dispatch(fetchUserActions(userId));
     },
+
+    logOpenUserActions() {
```

Comment:
Please add a new unit test for this new method, you can find examples testing other methods of the controller in the `headless-user-action.test.ts` file.

Reply:
added test